### PR TITLE
feat: add stats in via instrumentation

### PR DESCRIPTION
**Pull request by elseagle**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -100,6 +104,14 @@ def process_import_function(event, context):
...
+                "last_collection_run_time",
+                {
+                    "jurisdiction": juris["name"],
+                    "scrape_type": "import",
+                },
+            )
+
         except Exception as e:
             logger.error(
                 f"Error importing jurisdiction {juris['id']}: {e}"
```
Comment (johnseekins) [inline app.py:116]: We want to force a batch send (`stats.close()`) before the lambda terminates or we'll miss some data. Also, we might consider a `objects_collected{jurisdiction=<>,session=<>}` counter to indicate how many objects are being collected this way. There is a similar metric in the regular ingest that you could use as reference.

Comment (elseagle) [reply app.py:116]: Thanks @johnseekins - this stat is still being collected by the scraper, that's why I didn't add it to the lambda, the stat added here is exempted in os-core when realtime is activated. If we do this we'd be duplicating the stat.

Comment (johnseekins) [reply app.py:116]: Okay. If stats are still being produced, that's great. We'll still want to send `stats.close()` to ensure all batched stats are sent at the end of the lambda function.

Comment (elseagle) [reply app.py:116]: Yeah, will add that now.

**Review by bfossen-ce — APPROVED** [review]
LGTM

**Pull request merged**